Makefile.system

```diff
@@ -954,7 +954,7 @@ endif
 ifeq ($(ARCH), loongarch64)
 LA64_ABI=$(shell $(CC) -mabi=lp64d -c $(TOPDIR)/cpuid_loongarch64.c -o /dev/null > /dev/null 2> /dev/null && echo lp64d)
 ifneq ($(LA64_ABI), lp64d)
-LA64_ABI=lp64
+LA64_ABI=lp64d
```

Comment:
sorry, this change is almost certainly incorrect - the code here actually checks if the compiler supports the lp64d ABI and falls back to the older lp64 name if the compiler is too old to accept -mabi=lp64d

Reply:
ok, done.